### PR TITLE
Adding examples and additional features to packed-with-themida.yml

### PR DESCRIPTION
**Pull request by jtothej**
This PR updates mbc category, adds examples and additional features to the packed-with-themida.yml rule

### CODE REVIEW

**Review by mr-tz — APPROVED** [review]

File: nursery/packed-with-themida.yml [hunk 1]
```diff
@@ -7,13 +7,20 @@ rule:
     att&ck:
       - Defense Evasion::Obfuscated Files or Information::Software Packing [T1027.002]
     mbc:
-      - Anti-Static Analysis::Software Packing [F0001]
+      - Anti-Static Analysis::Software Packing::Themida [F0001.011]
     references:
       - https://www.hexacorn.com/blog/2016/12/15/pe-section-names-re-visited/
     examples:
+      - 8a132663bee5c2f0f5cbfebee1b55ac72934632bf32bc32d6e2dae797c9e6e35
+      - 2826b762b9c268601a44974ef469a671b441e798a6c3cbb40070450c6c030ba2
```
Comment (williballenthin) [inline nursery/packed-with-themida.yml:15]: now that we have examples, would you move this file out of the nursery?

File: nursery/packed-with-themida.yml [hunk 2]
```diff
@@ -7,13 +7,20 @@ rule:
...
   features:
     - or:
       - section: Themida
       - section: .Themida
       - section: .themida
       - section: WinLicen
+      - section: .winlice
+      - count(section(        )): 2 or more
```
Comment (williballenthin) [inline nursery/packed-with-themida.yml:23]: can you add a description or comment here describing the whitespace? is it 8 spaces?

**Review by williballenthin — CHANGES_REQUESTED** [review]
with these examples, we can graduate the rule, so lets do that

File: nursery/packed-with-themida.yml (hunk 1 above)
Comment (jtothej) [reply nursery/packed-with-themida.yml]: I moved the rule from nursery to anti-analysis/packer/themida

File: nursery/packed-with-themida.yml (hunk 2 above)
Comment (jtothej) [reply nursery/packed-with-themida.yml]: Sure, I added relevant descriptions

Comment (williballenthin) [reply nursery/packed-with-themida.yml]: perfect, thanks!

**Review by williballenthin — APPROVED** [review]
thank you!

**Pull request merged**